R/df_to_array.R

```diff
@@ -8,7 +8,7 @@
 #'
 #' @keywords internal
 #'
-df.to.array <- function(
+df.to.array <- function( # nolint: object_name_linter
```

Comment:
we skip linting the object name because it is the deprecated version of the renamed function, and thus cannot be renamed here.